extensions/wikia/InsightsV2/services/InsightsService.class.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+
+class InsightsService extends WikiaService{
+
+	/**
+	 * @param string $type type of model
+	 * @param int $size number of pages we need
+	 * @return array
+	 */
+	public function getInsightPages( $type, $size, $sortingType ) {
+		if ( !InsightsHelper::isInsightPage( $type ) ) {
+			return [];
+		}else{
```

Comment:
You have a return in the `if` block so it would be cleaner to remove the `else` and just unindent the main body of the method.